tests/profiler/user_workflow_fixtures.py

```diff
@@ -0,0 +1,194 @@
+from typing import List
+
+import pytest
+
+from great_expectations.core import ExpectationConfiguration, ExpectationSuite
+
+# TODO: Move these fixtures to integration tests
+
+@pytest.fixture(scope="module")
+def alice_columnar_table_single_batch():
+    profiler_config = """
+variables:
+  max_user_id: 999999999999
+  min_timestamp: 2004-10-19 10:23:54
+rules:
+  my_rule_for_user_ids:
+    domain_builder:
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      semantic_types:
+        - user_id
```

Comment:
We currently do not have a "user_id" semantic type -- we have "identity"... Or is this somehow inferred?..

Reply:
This would be a user defined semantic type. Maybe it is more clear to change the class_name to something other than SimpleSemanticTypeColumnDomainBuilder, what do you think?

Reply:
Changed! Let me know what you think.